src/components/freelancer/project/projectProfileDetailCard.tsx

```diff
@@ -61,26 +61,26 @@ export function ProjectProfileDetailCard({
   startDate,
   endDate,
   className,
-  domain_id,
+  // domain_id,
   ...props
 }: CardProps) {
   const [amount, setAmount] = useState('');
   const [descriptionValue, setDescription] = useState('');
   const [dialogOpen, setDialogOpen] = useState(false);
-  const user = useSelector((state: RootState) => state.user);
-  const params = useParams();
+  // const user = useSelector((state: RootState) => state.user);
+  // const params = useParams();
 
   const handleSubmit = async (e: any) => {
     e.preventDefault();
 
     try {
-      const response = await axiosInstance.post(`/bid`, {
-        current_price: amount,
-        description: descriptionValue,
-        bidder_id: user.uid,
-        project_id: params.project_id,
-        domain_id: domain_id,
-      });
+      // const response = await axiosInstance.post(`/bid`, {
```

Comment:
Why are we removing API call here?

Reply:
sir fixing lint issue .....

Reply:
this is a post request, so there is a possibility that its response is not used anywhere, does not mean you should remove api call alltogether.
I am not sure if this post is required here or not @Akhil-chaudhary please advise if this was needed to be removed

If api is required here then there are 2 possible ways to fix this lint issue

1. Don't take response in the variable, just call ' await axiosInstance.post(`/bid`, {' directly without 'const response'
2. Better approach => there should be a sanity check of the api response to see if your request was processed correctly. Ideally there should be a status code or message you can check inside api response. This will fix lint issue and also make sure your post was succesful

Reply:
Actually  sir I don't know who is adding that post request  my task is to design the card but when it showing warning then just commit it ......I have not any idea about that post.